app/services/scenes/scenes.ts

```diff
@@ -189,67 +184,57 @@ export class ScenesService extends StatefulService<IScenesState> implements ISce
   }
 
   get scenes(): Scene[] {
-    return this.state.displayOrder.map(id => {
-      return this.getScene(id);
-    });
+    return uniqBy(this.state.displayOrder.map(id => this.getScene(id)), x => x.id);
   }
 
-
   get activeSceneId(): string {
     return this.state.activeSceneId;
   }
 
-
   get activeScene(): Scene {
     return this.getScene(this.state.activeSceneId);
   }
 
   suggestName(name: string): string {
     return namingHelpers.suggestName(name, (name: string) => {
-      const ind = this.activeScene
-        .getNodes()
-        .findIndex(node => node.name === name);
+      const ind = this.activeScene.getNodes().findIndex(node => node.name === name);
       return ind !== -1;
     });
   }
 
-
-  showNameScene(options: {rename?: string, itemsToGroup?: string[] } = {}) {
+  showNameScene(options: { rename?: string; itemsToGroup?: string[] } = {}) {
     this.windowsService.showWindow({
       componentName: 'NameScene',
       title: options.rename ? $t('Rename Scene') : $t('Name Scene'),
       queryParams: options,
       size: {
         width: 400,
-        height: 250
-      }
+        height: 250,
+      },
     });
   }
 
-
-  showNameFolder(options: { renameId?: string, itemsToGroup?: string[], parentId?: string } = {}) {
+  showNameFolder(options: { renameId?: string; itemsToGroup?: string[]; parentId?: string } = {}) {
     this.windowsService.showWindow({
       componentName: 'NameFolder',
       title: options.renameId ? $t('Rename Folder') : $t('Name Folder'),
       queryParams: options,
       size: {
         width: 400,
-        height: 250
-      }
+        height: 250,
+      },
     });
   }
 
-
   showDuplicateScene(sceneName: string) {
     this.windowsService.showWindow({
       componentName: 'NameScene',
       title: $t('Name Scene'),
       queryParams: { sceneToDuplicate: sceneName },
       size: {
         width: 400,
-        height: 250
-      }
+        height: 250,
+      },
     });
   }
 }
```

Comment:
it doesn't look like anything was actually changed in this file apart from style changes? i think we should keep those in separate PRs to reduce diff size and cognitive load when reviewing